icepyx/core/granules.py

```diff
@@ -165,9 +180,16 @@ def get_avail(self, CMRparams, reqparams):
 
             # print(results)
 
-            if len(results["feed"]["entry"]) == 0:
-                # Out of results, so break out of loop
-                break
+            try:
+                if len(results["feed"]["entry"]) == 0:
```

Comment:
FWIW I think Pep 8 prefers: `if not results["feed"]["entry"]:`

Reply:
Thanks for pointing that out!